cabal.project

```diff
@@ -95,8 +95,10 @@ allow-newer:
 source-repository-package
    type: git
    location: https://github.com/IntersectMBO/ouroboros-network
-   tag: b07a86ed853b63881b5a83e57508902f1562ac01
-   --sha256: sha256-n/XX0+cQegq2a1cAfmGx30T64eix4oEXzpVEFCKqmg0=
+   tag: 0db8669b67982cba755e80bf2e413527def41244
+   --sha256: sha256-vEO721Xab0RTVKFQFKal5VCV5y+OUzELo8+7Z8TETJQ=
    subdir:
+     ouroboros-network
+     ouroboros-network-protocols
```

Comment:
https://github.com/IntersectMBO/ouroboros-network/pull/5202 has not been merged yet, so the comment above needs to be updated